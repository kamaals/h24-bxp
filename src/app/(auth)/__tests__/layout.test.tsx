import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import AuthLayout from "../layout";
import { LogIn } from "lucide-react";

describe("Auth Layout", () => {
  it("Should render AuthLayout Correctly", () => {
    const result = render(
      <AuthLayout>
        <LogIn />
      </AuthLayout>
    );
    expect(result).toMatchSnapshot();
  });
});
