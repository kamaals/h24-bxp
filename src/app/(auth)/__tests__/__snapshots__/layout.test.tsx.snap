// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Auth Layout > Should render AuthLayout Correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="flex container items-center justify-center mx-auto max-w-sm"
      >
        <svg
          aria-hidden="true"
          class="lucide lucide-log-in"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15 3h4a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2h-4"
          />
          <polyline
            points="10 17 15 12 10 7"
          />
          <line
            x1="15"
            x2="3"
            y1="12"
            y2="12"
          />
        </svg>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="flex container items-center justify-center mx-auto max-w-sm"
    >
      <svg
        aria-hidden="true"
        class="lucide lucide-log-in"
        fill="none"
        height="24"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M15 3h4a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2h-4"
        />
        <polyline
          points="10 17 15 12 10 7"
        />
        <line
          x1="15"
          x2="3"
          y1="12"
          y2="12"
        />
      </svg>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
